docs/static/schemas/1-0-0/component.schema.json

```diff
@@ -2,7 +2,7 @@
   "$id": "/1-0-0/component.schema.json",
   "$schema": "https://json-schema.org/draft/2020-12/schema",
   "title": "Dynamic Component Description",
-  "description": "A full description of a dynamic component in the AICA application framework.",
+  "description": "A full description of a dynamic component in the AICA System.",
```

Comment:
IMO all the changes in the `static/schemas` folder are not necessary, since we don't need to backport these changes IMO. We will just have to make sure that the new schema versions are correct

Reply:
thanks, indeed manual changes to `static` directory are technically prohibited, since these are hosting our historical versioned fixtures and schemas. Even innocuous changes like this might cause weird backwards compatibility issues for an application that relies on this document at this URL (very very unlikely but still possible). In short, old releases should not be updated